packages/utils/odsp-doclib-utils/src/odspAuth.ts

```diff
@@ -13,6 +13,13 @@ export interface IOdspTokens {
     readonly refreshToken: string;
 }
 
+/**
+ * Configuration for a public client.
+ */
+export interface IPublicClientConfig {
```

Comment:
typetests don't support removing interfaces on LTS.

Reply:
Deprecate it? Not that it matters

Reply:
yeahhh this is all internal in modern code anyway, so probably we did that at some point.